polkadot/xcm/pallet-xcm/src/precompiles.rs

```diff
@@ -81,7 +82,7 @@ where
 					final_destination.into(),
 					final_message.into(),
 				)
-				.map(|message_id| message_id.encode())
+				.map(|_| RETURN_VALUE.encode())
```

Comment:
Why not this? Seems simpler.

```suggestion
				.map(|_| Vec::new())
```

Reply:
Addressed on https://github.com/paritytech/polkadot-sdk/pull/9135